config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-release-0.1-config.yaml

```diff
@@ -0,0 +1,37 @@
+periodics:
+- interval: 12h
+  name: periodic-secrets-store-csi-driver-upgrade-test-azure-release-0-1
```

Comment:
some more description on the intended purpose of this test would help. either in comment or just as PR description.

from waht I understand it tries installing the current published release wiht helm then uses the manifest_staging helm update on the release-0.1 branch.

I guess we would keep this around for as long as we plan to support on 0.1.x?

Reply:
That's right! Updated the PR description.